src/nvim/tag.c

```diff
@@ -975,7 +975,8 @@ add_llist_tags(
             if (cmd_len > (CMDBUFFSIZE - 5)) {
                 cmd_len = CMDBUFFSIZE - 5;
             }
-            xstrlcat((char *)cmd, (char *)cmd_start, cmd_len);
+            snprintf((char *)cmd + len, CMDBUFFSIZE + 1 - len,
```

Comment:
You could have used  `STRNCAT` like Vim. The macro is used in other files.

Reply:
Thanks for reviewing and merging.  Oh wow.. you're right about STRNCAT.  I thought for sure it was deprecated and would fail lint. 🤦